ci-operator/step-registry/ipi/conf/azure/custom-region/ipi-conf-azure-custom-region-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: ipi-conf-azure-custom-region
+  from_image:
+    namespace: ocp
+    name: "4.11"
```

Comment:
I am not sure if this is correct @jianlinliu

Reply:
In the step, it mainly use `yq-go`  in the upi-installer image, so I do not care about the version tag.

Reply:
Thanks @sunzhaohua2 and @jianlinliu